tests/python_package_test/test_dask.py

```diff
@@ -171,6 +223,32 @@ def test_regressor(output, client, listen_port):
     assert_eq(y, p2, rtol=1., atol=50.)
 
 
+@pytest.mark.parametrize('output', data_output)
+def test_regressor_pred_contrib(output, client, listen_port):
+    X, y, w, dX, dy, dw = _create_data('regression', output=output)
+
+    dask_regressor = dlgbm.DaskLGBMRegressor(
```

Comment:
Add `n_estimators=10` and `num_leaves=10`?
#3786.

Reply:
yep! sorry about the bad merge conflict resolution